data/systemd/snapd.apparmor.service.in

```diff
@@ -17,6 +17,3 @@ RequiresMountsFor=/var/cache/apparmor /var/lib/snapd/apparmor/profiles
 Type=oneshot
 ExecStart=@libexecdir@/snapd/snapd-apparmor start
 RemainAfterExit=yes
-
-[Install]
-WantedBy=multi-user.target
```

Comment:
we want to ensure as much as possible that the service loaded the profile before any user app are started, it's not only about snapd itself, so I'm not sure I understand this change

Reply:
WantedBy is only to pull jobs in the dependencies. But it does not say when. After/Before is what is used in systemd to order the jobs. So that we make multi-user.target or snapd.service want snapd.apparmor.service, it does not change the order in which snapd.apparmor.service starts.

Reply:
Maybe you mean if someone disables snapd.service, but not the apps installed by snapd? Right. It sounds to me that the generated service files should maybe have a Wants= for some target that wants snapd.apparmor.service.

Reply:
I have done some refactoring of the dependencies introducing `snapd.system-services-pre.target`